src/OpenTelemetry.Instrumentation.MassTransit/README.md

```diff
@@ -48,6 +48,27 @@ services.AddOpenTelemetrySdk(x =>
 });
 ```
 
+Instrumentation can be configured via options overload for
+`AddMassTransitInstrumentation` method:
+
+```csharp
+using var openTelemetry = Sdk.CreateTracerProviderBuilder()
+    .AddMassTransitInstrumentation(options =>
+    {
+        // Enable enriching an activity after it is created.
+        options.Enrich = (activity, eventName, rawObject) =>
```

Comment:
An example that actually makes use of the rawObject passed would be more helpful.